docs/backport_binary.md

```diff
@@ -6,14 +6,27 @@ looks like to pin fleet to the version currently provided by GiantSwarm. Simply
 set the following flags to your `yochu` executable.
 
 ```
---fleet-version=v0.11.5-gs-grpc-1 --steps=distribution,overlay,fleet
+--http-endpoint=https://downloads.giantswarm.io --fleet-version=v0.11.5-gs-grpc-1 --steps=distribution,overlay,fleet
 ```
 
+The `--http-endpoint` defaults to `https://downloads.giantswarm.io`. We are
+using it here for completeness. This can be any other location though. The
+expected locations for `fleet` version `v0.11.5-gs-grpc-1` would be the
+following.
+
+- `https://downloads.giantswarm.io/fleet/v0.11.5-gs-grpc-1/fleetd`
+- `https://downloads.giantswarm.io/fleet/v0.11.5-gs-grpc-1/fleetctl`
+
+See:
+ * https://github.com/giantswarm/yochu/blob/master/cli/setup_cmd.go#L36
+ * https://github.com/giantswarm/yochu/blob/master/steps/fleet/fleet.go#L34
+
 Since naming is hard here a short explanation of the steps.
 - Step `distribution` step prepares the readonly file system on CoreOS by
   creating certain directories.
```

Comment:
Right, I got the explanation a little bit wrong. I wanted to point out that step distribution creates a directory and that this step takes care of doing so within the readonly file system of CoreOS. I updated it.